wnprc_billing/resources/web/wnprc_billing/form/field/MiscChargesDebitAcctField.js

```diff
@@ -0,0 +1,5 @@
+Ext4.define('WNPRC_Billing.form.field.MiscChargesDebitAcctField', {
+    extend: 'Ext.form.field.Text',
```

Comment:
Do we need to define new classes for this and TotalCostField, or can we just apply the same properties in the single place that uses in MiscCharges.js?

Reply:
- TotalCostField is used in both form configs MiscCharges.js (charges with animal ids) and also in NonAnimalCharges.js (charges without animalIds), hence a new common class so that its field properties are not duplicated.

- MiscChargesDebitAcctField.js is specifically for MiscCharges.js (charges with animal ids) - it is non-editable and derived from ehr.project.account (as per client request). While the one used in NonAnimalCharges.js (charges without animalIds) is sourced from ehr_billing.aliases. So different configs & cases for each forms.